extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.css

```diff
@@ -115,20 +115,31 @@
   white-space: nowrap !important;
 }
 
-.i-amphtml-lbv[gallery-view] .i-amphtml-lbv-gallery {
-  display: flex;
-  flex-wrap: wrap;
-  align-content: flex-start;
+@media screen and (min-width: 1024px) {
+  .i-amphtml-lbv[gallery-view] .i-amphtml-lbv-gallery {
+    display: grid;
+    grid-template-columns: repeat(4, 246px);
```

Comment:
where does `246px` come from?

Reply:
`(1024 px - 4 * 10px) / 4`, where 10 px is the grid gap plus 5 px of whitespace for the left and right border.

Reply:
I originally wanted to make it 25%, but that doesn't leave out the 5 px of whitespace that we want for the external borders, unless I want to shove a `calc` block inside the repeat block for `calc((1024 px - 4 * 10px) / 4)`.

Reply:
We use `postcss` and `cssnano` in our build chain, which should simplify `calc` when possible. It'd be good to verify that, but having the full `calc` is more maintainable than just the result that looks like a magic number.

Reply:
In that case, I'll make this calculation a bit more explicit.